lab/src/Microsoft.ServiceModel.Syndication/src/Atom10FeedFormatter.cs

```diff
@@ -763,155 +868,160 @@ private SyndicationContent ReadContentFrom(XmlReader reader, SyndicationItem ite
                             }
                             else
                             {
-                                TraceSyndicationElementIgnoredOnRead(reader);
+                                result.AttributeExtensions.Add(new XmlQualifiedName(reader.LocalName, reader.NamespaceURI), await reader.GetValueAsync());
                             }
```

Comment:
Why are we saving the attribute extension when _preserveAttributeExtensions is false?